feat(curriculum): styling checkboxes workshop (#62607)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-game-settings-panel/68e9a6201e2efa142d62bfe1.md

---
id: 68e9a6201e2efa142d62bfe1
title: Step 7
challengeType: 0
dashedName: step-7
---

# --description--

Next up, you are going to enlarge the checkboxes for better visibility.

You will begin by setting up an attribute selector for `input`, but specifically targetting your `[type="checkbox"]`.

Within your attribute selector you will want to set the `width` and the `height` to `20px`. This makes it larger than it was before.

And to conform with your `cursor` setting that was set on the labels you are going to add `cursor` and assign the value of `pointer` to it. Now when you hover over the labels it will display a pointer.

# --hints--

You should have an attribute selector in your CSS of `input[type="checkbox"]`

```js
assert.exists(new __helpers.CSSHelp(document).getStyle('input[type="checkbox"]'));
```

You should add `width` to your `input[type="checkbox]` attribute selector.

```js
assert.isOk(new __helpers.CSSHelp(document).getStyle('input[type="checkbox"]')?.width);
```

> You should have `width` with a value of `20px`.

Reviewer: ```suggestion
You should have a `width` property with a value of `20px`.
```

```js
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('input[type="checkbox"]')?.width, "20px");
```

> You should add `height` to your `input[type="checkbox]` attribute selector.

Reviewer: ```suggestion
You should add a `height` property to your `input[type="checkbox]` selector.
```

```js
assert.isOk(new __helpers.CSSHelp(document).getStyle('input[type="checkbox"]')?.height);
```

You should have `height` with a value of `20px`.

```js
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('input[type="checkbox"]')?.height, "20px");
```

You should add `cursor` to your `input[type="checkbox]` attribute selector.

```js

> assert.isOk(new __helpers.CSSHelp(document).getStyle('input[type="checkbox"]')?.cursor);

Reviewer: ```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('input[type="checkbox"]')?.cursor);
```

```

> You should have `cursor` with a value of `pointer`.

Reviewer: ```suggestion
You should have a `cursor` property with a value of `pointer`.
```
